sql/moz-fx-data-shared-prod/firefox_ios_derived/attributable_clients_v1/query.sql

```diff
@@ -0,0 +1,144 @@
+CREATE TEMP FUNCTION sum_map_values(map ARRAY<STRUCT<key STRING, value INT64>>)
+RETURNS INT64 AS (
+  (SELECT SUM(value) FROM UNNEST(map))
+);
+
+WITH client_day AS (
+  SELECT
+    DATE(submission_timestamp) AS submission_date,
+    sample_id,
+    client_info.client_id,
+    LOGICAL_OR(
+      mozfun.norm.extract_version(client_info.app_display_version, 'major') >= 107
+    ) AS has_search_data,
+    SUM(sum_map_values(metrics.labeled_counter.search_in_content)) AS searches,
```

Comment:
I don't think the `baseline` ping in ios has search metrics. That may be an android-only thing. If so you can get it just from the `mobile_search_clients_daily_v1` table

Reply:
also, I'm not sure we should be excluding this. It looks like for versions 107 and below we need to get this information from baseline and only after this version we can retrieve it from `mobile_search_clients_daily_v1`. Maybe @bani can confirm?

Reply:
@kik-kik my understanding is the same as Frank's: for iOS we should use mobile_search_clients_daily_v1 only.

For context, our historical source of truth for searches both on Android and iOS is `mobile_search_clients_daily_v1`, which reads from the metrics ping. But we were having some issues with ping joining and missing data due to the metrics ping schedule, and that was a problem for our current work on LTV model for Android, so Perry added searches to the baseline ping on Fenix a few months ago to help with that moving forward.

Reply:
ok, I hope I understood correctly your comments. I adjusted the query and completely removed the baseline table from the query.